opentelemetry-api/src/opentelemetry/distributedcontext/__init__.py

```diff
@@ -11,3 +11,114 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import abc
+from contextlib import contextmanager
+import string
+import typing
+
+
+PRINTABLE = set(string.printable)
+
+
+class EntryMetadata(dict):
```

Comment:
Is there any reason to require this class to inherit from `dict`? The spec says:

> EntryMetadata contains properties associated with an Entry. For now only the property EntryTTL is defined. In future, additional properties may be added to address specific situations.

In Java for example this is defined as an standard class with a single property.